examples/inference/README.md

```diff
@@ -0,0 +1,125 @@
+# Colossal-Inference
+
+
+## Table of Contents
+
+-   📚 Introduction
+
+- 🔨 Installation
+- 🚀 Quick Start
+- 💡Usage
+
+## 📚 Introduction
+
+This example lets you to set up and quickly try out our Colossal-Inference.
+
+## 🔨 Installation
+
+### Install From Source
+
+Prerequistes:
+
+-   Python == 3.9
+-   PyTorch >= 2.1.0
+-   CUDA == 11.8
+-   Linux OS
+
+We strongly recommend you use [Anaconda](https://www.anaconda.com/) to create a new environment (Python >= 3.9) to run our examples:
+
+```shell
+# Create a new conda environment
+conda create -n inference python=3.9 -y
+conda activate inference
+```
+
+Install the latest PyTorch (with CUDA == 11.8) using conda:
+
+```shell
+conda install pytorch torchvision torchaudio pytorch-cuda=11.8 -c pytorch -c nvidia
+```
+
+Install Colossal-AI from source:
+
+```shell
+# Clone Colossal-AI repository to your workspace
+git clone https://github.com/hpcaitech/ColossalAI.git
+cd ColossalAI
+
+# Install Colossal-AI from source
+CUDA_EXT=1 pip install .
```

Comment:
You can remove the CUDA_EXT=1 so make installation easier. Any cuda extension will be automatically installed during runtime if needed.

Reply:
Fixed.